cordova/cordova-webapp-android-tests/webapp/webapp_multiple_apks_false.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
```

Comment:
I think these two tests should belong to cordova-feature-android-tests test suite.